router/core/modules.go

```diff
@@ -67,6 +70,32 @@ func RegisterModule(instance Module) {
 	modules[string(mod.ID)] = mod
 }
 
+// Sorts the modules by priority
+func SortModules(modules map[string]ModuleInfo) []ModuleInfo {
```

Comment:
We can make private if it is only used in core.

Reply:
Used in tests